src/core/Link/Link.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+import { useStyles } from './styles';
+import { BaseLinkProps } from './base';
+
+interface LinksProps extends BaseLinkProps {
+  pathname: string;
+  disabled?: boolean;
+  active?: boolean;
+}
+
+const Links: React.FC<LinksProps> = ({
+  pathname,
```

Comment:
here also camelCase

Reply:
@AVRahul @S-ayanide So, I've used the same pathname as it is in the props above